apps/molgenis-components/src/client/client.ts

```diff
@@ -82,7 +82,8 @@ export default {
           properties,
           schemaMetaData,
           myAxios,
-          schemaNameCache
+          schemaNameCache,
+          1
```

Comment:
que ? , '1'

Reply:
you could have checked the variable name function that is being called. 1 means '1 level deep' which is what the current default is; but if you increase this you will be able to expand the graphql to expand refs with N levels.

Reply:
suggest to use a named constant